src/mask.rs

```diff
@@ -70,8 +70,27 @@ impl TrieMask {
         self.0 == 0
     }
 
+    /// Returns the number of bits set in the mask.
+    #[inline]
+    pub const fn count_bits(self) -> u8 {
+        self.0.count_ones() as u8
+    }
+
+    /// Returns the index of the first bit set in the mask.
+    #[inline]
+    pub const fn first_set_bit_index(self) -> u8 {
+        self.0.trailing_zeros() as u8
+    }
```

Comment:
should this return `Option`? no bits might be set

Reply:
yep right, fixed